scss/bitstyles/atoms/skip-link/skip-link.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Atoms/Skip link" />
+
+# Skip link
```

Comment:
What do I have to do, to actually "see" this in Storybook? Maybe we should render this story differently 🤔

Reply:
Aha, ok maybe we should add a note above the example — you need to focus the link, otherwise it won't show. That means clicking inside the story then hitting tab so the focus goes to the link